.gitignore

```diff
@@ -7,6 +7,9 @@
 *.objs*
 *.d
 *.o
+*.gch
+# PCH generated by build
+source/Irrlicht/server_pch.h
```

Comment:
@Crisspl fix please.

Reply:
https://github.com/buildaworldnet/IrrlichtBAW/blob/master/source/Irrlicht/CMakeLists.txt#L376

Reply:
IrrlichtServer has to make its own pch because of different flags etc. However maybe it could be better to have in repo pch_server.h with just `#include "pch.h"` instead of making cmake creating it